src/main/java/seedu/address/logic/Logic.java

```diff
@@ -47,4 +47,5 @@ public interface Logic {
      * Set the user prefs' GUI settings.
      */
     void setGuiSettings(GuiSettings guiSettings);
+
```

Comment:
is this an extra new line that is not supposed to be here but checkstyles just missed it